src/main/java/com/at/avro/AvroField.java

```diff
@@ -47,6 +52,14 @@ public Object getDefaultValue() {
     public boolean isDefaultValueSet() {
         return defaultValue != NOT_SET;
     }
+    
+    public String getDoc() {
+        return doc;
+    }
+    
+    public boolean isDocSet() {
+        return doc != null;
```

Comment:
I think this is always true (based on test cases). Does column.getRemarks() return an empty string when there's no comment? I think we can change it to StringUtils.isNotBlank(doc). WDYT?

Reply:
Yes it does return an empty string when comments are missing.

My first thought was to always have the `doc` field, even an empty one, when flag `useSqlCommentsAsDoc` was set to `true`. However as the `doc` field is an optional one in an Avro schema, I could add it only when a comment is present.

Reply:
Replaced with a call to `Stringutils.isNotBlank`, the `doc` field will be added only when the flag `useSqlCommentAsDoc` is set to `true` and when there exists a non empty comment on the column.

Reply:
Thanks!